### DISCUSSION TITLE
Question about known issue in 4.3.5 version

### DISCUSSION THREAD

**Discussion by roysG** [emqx/emqx#11519] [Q&A]
In the past 1.5 years we use in version 4.3.5 as clustering of of 3 nodes with mongo authentication 
now we are going to add more nodes from other region.
The version seems as stable.

Is there any known issue with clustering/lag of this version?

**Comment by kjellwinblad** [comment]
It depends on factors such as how many MQTT clients you have, number of topics, frequency of publication and so forth if scalability will be a problem for you with version 4.3.5.

That being said, EMQX 5.X has been designed from the ground up to be more scalable than 4.X. Here is a write up explaining a little bit about how clustering is designed in 5.X and what type of load it can handle:

https://www.emqx.com/en/blog/reaching-100m-mqtt-connections-with-emqx-5-0

You are also saying that you are planning to add nodes from other region. I think this probably means that latency between the nodes will increase? I'm not sure how our clustering might be impacted by high latency between nodes so I will ask one of EMQX's clustering experts to answer that.

**Comment by roysG** [comment]
Hi,
Any update about the latency question?

בתאריך יום ו׳, 25 באוג׳ 2023 ב-11:00 מאת Kjell Winblad <
***@***.***>:

**Comment by id** [comment]
Hi @roysG.

Multi-region clusters based on 4.3 EMQX version most certainly will have higher latency on connect and subscribe operations. We recommend to do a performance test based on your traffic patterns first.
